### PR TITLE
Feature/3449 three liner examples

### PR DESCRIPTION
**Pull request by emclaug2**
#### Changes proposed in this Pull Request:
- Add Threeliner examples, playground & docs.

### CODE REVIEW

**Comment by huayunh** [comment]
I don't know what feedback you are looking for through this draft pull request, if any. I encourage you to communicate that in the PR description.

In the meantime, I recommend adding a **_simple_** toolbar menu to the "ThreeLiner with Custom Content" as a frequently seen use case. I also encourage you to write "ThreeLiner" in Title Case.

**Comment by emclaug2** [comment]
Hey @huayunh this was _supposed_ to be a real PR, not a draft!  My mistake - will update based on your comments.

File: src/docs/pages/component-docs/components/three-liner/three-liner-doc.module.ts [hunk 1]
```diff
@@ -1,16 +1,27 @@
 import { NgModule } from '@angular/core';
 import { BasicExample } from './examples/basic.component';
 import { SharedCompDocsModule } from '../../shared/shared-comp-docs.module';
-import { ChannelValueModule, ThreeLinerModule } from '@brightlayer-ui/angular-components';
+import { ChannelValueModule, ThreeLinerModule, ToolbarMenuModule } from '@brightlayer-ui/angular-components';
```
Comment (huayunh) [inline src/docs/pages/component-docs/components/three-liner/three-liner-doc.module.ts:4]: So it looks like you replaced channel value with toolbar menu, which is fine by me. Would you want to take `ChannelValueModule` out of the import statement?

Comment (emclaug2) [reply src/docs/pages/component-docs/components/three-liner/three-liner-doc.module.ts]: Good catch, it's removed now.

**Review by huayunh — APPROVED** [review]

**Comment by emclaug2** [comment]
🚀

**Pull request merged**